default/python/AI/AIDependencies.py

```diff
@@ -645,6 +645,11 @@
     "GREAT":    {},
     "ULTIMATE": {},
 }
+
+LAUNCH_BAY_BASE_CAPACITY = 2
+HANGAR_LAUNCH_CAPACITY_MODIFIER_DICT = {
+    "FT_HANGAR_1": 2,
```

Comment:
The hangar FOCS script would have no effect for a newly added launch bay. To make the AI aware of this, use a nested dict like
```
HANGAR_LAUNCH_CAPACITY_MODIFIER_DICT = {
    # hangar_name: {bay_name: effect, bay_name2: effect, ...}
    "FT_HANGAR_1": {"FT_BAY_1": 2}, 
}
```

and query via ```HANGAR_LAUNCH_CAPACITY_MODIFIER_DICT.get(hangar_name, {}).get(bay_name, 0)```

Reply:
Ok, finally i got you. I'd prefer giving an added part automatically the interceptor bonus, but as i see no easy way to do that, I'll take your code snippets, thank you